### ISSUE TITLE
Release version 3.4.1 to pypi

### ISSUE THREAD

**Issue by tfluehmann** [DinoTools/python-ssdeep#63]
##### ISSUE TYPE
- Bug Report

##### Python ssdeep VERSION
```
3.4.1
```

##### OS / ENVIRONMENT
*

##### SUMMARY
Release Version 3.4.1 to pypi

##### EXPECTED RESULTS
It would be nice if ssdeep release version 3.4.1 would be available on pypi as well.

**Comment by eljeffeg** [comment]
Yes, please push up the latest.  I've tried to install ssdeep (from PyPi) on two separate machines recently and it's failed on both.
